include/MiniFB_ex_enums.h

```diff
@@ -0,0 +1,162 @@
+#pragma once
+
+typedef enum { eFalse = 0, eTrue = 1 } eBool;

```

Comment:
I don't think `eBool` is needed https://en.wikibooks.org/wiki/C_Programming/stdbool.h can be used for C instead

Reply:
Ok, better. I thought that you wanted to maintain compatibility with older C versions